modules/quadbin/snowflake/sql/QUADBIN_KRING.sql

```diff
@@ -2,12 +2,17 @@
 -- Copyright (C) 2022 CARTO
 ----------------------------
 
--- FIXME: slow
+-- The function returns a STRING for two main issues related with Snowflake limitations
```

Comment:
I would move this code to the module's readme

Reply:
I'm not pretty sure about this because it's an implementation detail, it should not be impact in the user experience either the user should act in a special way in order to execute the functions.

Reply:
I'm not pretty sure about this because it's an implementation detail, it should not be impact in the user experience either the user should act in a special way in order to execute the functions.